### DISCUSSION TITLE
Add new reference page in the docs

### DISCUSSION THREAD

**Discussion by adevade** [spatie/ray#232] [Ideas]
I really love this page in the Livewire docs: https://laravel-livewire.com/docs/2.x/reference .
Just a list of available functions with short descriptions. Easy to scan!

I think it would really suit the Ray docs!

**Comment by freekmurze** [comment]
That's a nice idea. @willemvb could you take a look?

**Comment by willemvb** [comment]
Had a first go at it: https://spatie.be/docs/ray/v1/usage/reference

**Reply by adevade** [reply]
Wow! Nice!!! Love it!
